instrumentation/servlet/servlet-2.2/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/v2_2/Servlet2HttpServerTracer.java

```diff
@@ -16,7 +16,7 @@ public static Servlet2HttpServerTracer tracer() {
 
   @Override
   protected String getInstrumentationName() {
-    return "io.opentelemetry.javaagent.servlet";
```

Comment:
These actually seem like a no-brainer to me - our artifact names are also e.g., `opentelemetry-javaagent-servlet-2.2`, the name of the library. And these are supposed to reflect the name of the library.

I understand the potential for confusion due to the lack of something like a `+` but I think it's just the nature of how instrumentation libraries are named.